movie-app/pages/index.js

```diff
@@ -0,0 +1,26 @@
+// const register = require('@babel/register').default;
+// register({ extensions: ['.ts','.tsx', '.js', '.jsx'] });
+// require('./express.js');
+
+import React, { useEffect } from 'react';
+import { useRouter } from 'next/router';
+import { getNavUrl } from '../src/constants/constants';
+import { connect } from 'react-redux';
+
+const IndexApp = props => {
+  const router = useRouter();
+
+  const url = `/search${getNavUrl(props.searchParams)}`;
+  useEffect(() => {
+    if (!router.route.includes('search')) router.push(url);
```

Comment:
Are you sure this check is needed? how can we reach this page with route other than `/`?

Reply:
done